File: 0125-valid-palindrome/0125-valid-palindrome.py
class Solution:
    def isPalindrome(self, s: str) -> bool:
        ans=""
        for i in range(0,len(s)):
            if(s[i].isalnum()):
                ans+=s[i].lower()
        res=ans[::-1]
        if(res==ans):
            return True
        return False

        

File: 0125-valid-palindrome/0125-valid-palindrome.cpp
class Solution {
public:
    bool isPalindrome(string s) {
        string sk="";
         for(char c : s){
            if(isalnum(c)){
                sk.push_back(tolower(c));
            }
        }
        string rev=sk;
        reverse(rev.begin(),rev.end());

       if(sk==rev){
        return true;
       }return false;
       
    }
};